drivers/wasapi/audio_driver_wasapi.cpp

```diff
@@ -37,6 +37,25 @@
 
 #include <functiondiscoverykeys.h>
 
+#ifndef PKEY_Device_FriendlyName
+
+#undef DEFINE_PROPERTYKEY
+#define DEFINE_PROPERTYKEY(id, a, b, c, d, e, f, g, h, i, j, k, l) \
+	const PROPERTYKEY id = { { a, b, c, {                          \
+												d,                 \
+												e,                 \
+												f,                 \
+												g,                 \
+												h,                 \
+												i,                 \
+												j,                 \
+												k,                 \
+										} },                       \
+		l };
+
```

Comment:
Why is this formatted like that?

Reply:
I had to format it like because of clang-format, it was failing otherwise.

Reply:
Thanks for the help Rémi!